src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/Sve.cs

```diff
@@ -815,6 +815,438 @@ public new abstract class Arm64 : AdvSimd.Arm64
+        /// </summary>
+        public static unsafe Vector<double> ConditionalExtractAfterLastActiveElement(Vector<double> mask, Vector<double> defaultValue, Vector<double> data) => ConditionalExtractAfterLastActiveElement(mask, defaultValue, data);
+
+        /// <summary>
+        /// svfloat64_t svclasta[_f64](svbool_t pg, svfloat64_t fallback, svfloat64_t data)
+        ///   CLASTA Ztied.D, Pg, Ztied.D, Zdata.D
+        /// float64_t svclasta[_n_f64](svbool_t pg, float64_t fallback, svfloat64_t data)
+        ///   CLASTA Xtied, Pg, Xtied, Zdata.D
+        ///   CLASTA Dtied, Pg, Dtied, Zdata.D
+        /// </summary>
+        public static unsafe double ConditionalExtractAfterLastActiveElement(Vector<double> mask, double defaultValues, Vector<double> data) => ConditionalExtractAfterLastActiveElement(mask, defaultValues, data);
+
+        /// <summary>
+        /// svint16_t svclasta[_s16](svbool_t pg, svint16_t fallback, svint16_t data)
+        ///   CLASTA Ztied.H, Pg, Ztied.H, Zdata.H
+        /// </summary>
+        public static unsafe Vector<short> ConditionalExtractAfterLastActiveElement(Vector<short> mask, Vector<short> defaultValue, Vector<short> data) => ConditionalExtractAfterLastActiveElement(mask, defaultValue, data);
+
+        /// <summary>
+        /// svint16_t svclasta[_s16](svbool_t pg, svint16_t fallback, svint16_t data)
+        ///   CLASTA Ztied.H, Pg, Ztied.H, Zdata.H
+        /// int16_t svclasta[_n_s16](svbool_t pg, int16_t fallback, svint16_t data)
+        ///   CLASTA Wtied, Pg, Wtied, Zdata.H
+        ///   CLASTA Htied, Pg, Htied, Zdata.H
+        /// </summary>
+        public static unsafe short ConditionalExtractAfterLastActiveElement(Vector<short> mask, short defaultValues, Vector<short> data) => ConditionalExtractAfterLastActiveElement(mask, defaultValues, data);
+
+        /// <summary>
+        /// svint32_t svclasta[_s32](svbool_t pg, svint32_t fallback, svint32_t data)
+        ///   CLASTA Ztied.S, Pg, Ztied.S, Zdata.S
+        /// </summary>
+        public static unsafe Vector<int> ConditionalExtractAfterLastActiveElement(Vector<int> mask, Vector<int> defaultValue, Vector<int> data) => ConditionalExtractAfterLastActiveElement(mask, defaultValue, data);
+
+        /// <summary>
+        /// svint32_t svclasta[_s32](svbool_t pg, svint32_t fallback, svint32_t data)
+        ///   CLASTA Ztied.S, Pg, Ztied.S, Zdata.S
+        /// int32_t svclasta[_n_s32](svbool_t pg, int32_t fallback, svint32_t data)
+        ///   CLASTA Wtied, Pg, Wtied, Zdata.S
+        ///   CLASTA Stied, Pg, Stied, Zdata.S
+        /// </summary>
+        public static unsafe int ConditionalExtractAfterLastActiveElement(Vector<int> mask, int defaultValues, Vector<int> data) => ConditionalExtractAfterLastActiveElement(mask, defaultValues, data);
+
+        /// <summary>
+        /// svint64_t svclasta[_s64](svbool_t pg, svint64_t fallback, svint64_t data)
+        ///   CLASTA Ztied.D, Pg, Ztied.D, Zdata.D
```

Comment:
Since we are generating [CLASTA (vectors)](https://docsmirror.github.io/A64/2023-06/clasta_z_p_zz.html) using `ConditionalExtractAfterLastActiveElementAndReplicate` APIs, this one should be [CLASTA (SIMD&FP scalar)](https://docsmirror.github.io/A64/2023-06/clasta_v_p_z.html) and hence the comment should reflect that. Perhaps the tool did not produce this summary docs correctly. CC: @a74nh, @swapnilgaikwad

Reply:
Done.